evals/test_cases/sql_tests/grouping_tests.py

```diff
@@ -5,71 +5,117 @@
 from evals.test_cases.sql_tests.constants import *
 
+TEST_TYPE = "aggregation_and_grouping"
+
 AGGREGATION_AND_GROUPING_TESTS = [
     # SMALL SCHEMA
     SQLTestCase(
         name="average_market_cap",
         user_input="What is the average market cap of all companies in the S&P 500?",
         schema=SMALL_SCHEMA,
         notebook_state=EMPTY_NOTEBOOK,
-        expected_output="SELECT AVG(MARKETCAP) FROM SP_500.PUBLIC.SP500_COMPANIES",
+        expected_output="""
+            SELECT AVG(MARKETCAP) 
+            FROM SP_500.PUBLIC.SP500_COMPANIES
+        """.strip(),
+        test_type=TEST_TYPE,
     ),
     SQLTestCase(
         name="average_market_cap_by_sector",
         user_input="What is the average market cap of companies by sector?",
         schema=SMALL_SCHEMA,
         notebook_state=EMPTY_NOTEBOOK,
-        expected_output="SELECT SECTOR, AVG(MARKETCAP) FROM SP_500.PUBLIC.SP500_COMPANIES GROUP BY SECTOR",
+        expected_output="""
+            SELECT SECTOR, AVG(MARKETCAP) 
+            FROM SP_500.PUBLIC.SP500_COMPANIES 
+            GROUP BY SECTOR
+        """.strip(),
+        test_type=TEST_TYPE,
     ),
     SQLTestCase(
         name="tech_companies_w_positive_revenue_growth",
         user_input="How many tech companies have positive revenue growth?",
         schema=SMALL_SCHEMA,
         notebook_state=EMPTY_NOTEBOOK,
-        expected_output="SELECT COUNT(*) FROM SP_500.PUBLIC.SP500_COMPANIES WHERE SECTOR = 'Technology' AND REVENUEGROWTH > 0",
+        expected_output="""
+            SELECT COUNT(*) 
+            FROM SP_500.PUBLIC.SP500_COMPANIES 
+            WHERE SECTOR = 'Technology' AND REVENUEGROWTH > 0
+        """.strip(),
+        test_type=TEST_TYPE,
     ),
     # MEDIUM SCHEMA
     SQLTestCase(
         name="avg_ltv_for_churned_customers",
         user_input="What is the average LTV for churned customers?",
         schema=MEDIUM_SCHEMA,
         notebook_state=EMPTY_NOTEBOOK,
-        expected_output="SELECT AVG(CLTV) FROM TELCO_CHRUN.PUBLIC.STATUS_ANALYSIS WHERE CHURN_LABEL = TRUE",
+        expected_output="""
+            SELECT AVG(CLTV) 
+            FROM TELCO_CHRUN.PUBLIC.STATUS_ANALYSIS 
+            WHERE CHURN_LABEL = TRUE
+        """.strip(),
+        test_type=TEST_TYPE,
     ),
     SQLTestCase(
         name="avg_referrals_for_all_services",
         user_input="What is the avg number of referrals for customer subscribed to all services?",
         schema=MEDIUM_SCHEMA,
         notebook_state=EMPTY_NOTEBOOK,
-        expected_output="SELECT AVG(NUMBER_OF_REFERRALS) FROM TELCO_CHRUN.PUBLIC.SERVICE_OPTIONS WHERE INTERNET_SERVICE = TRUE AND PHONE_SERVICE = TRUE",
+        expected_output="""
+            SELECT AVG(NUMBER_OF_REFERRALS) 
+            FROM TELCO_CHRUN.PUBLIC.SERVICE_OPTIONS 
+            WHERE INTERNET_SERVICE = TRUE AND PHONE_SERVICE = TRUE
+        """.strip(),
+        test_type=TEST_TYPE,
     ),
     SQLTestCase(
         name="avg_referrals_for_all_services",
```

Comment:
I think this ie misnamed. It's not related to referrals. I think just copy and pasted from last one